embulk-output-redshift/build.gradle

```diff
@@ -31,3 +31,62 @@ embulkPlugin {
     category = "output"
     type = "redshift"
 }
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            pom {  // https://central.sonatype.org/pages/requirements.html
+                developers {
+                    developer {
+                        name = "Sadayuki Furuhashi"
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Hitoshi Tanaka"
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "Hiroyuki Sato"
+                        email = "hiroysato@gmail.com"
+                    }
+                    // developer {  // https://github.com/embulk/embulk-output-jdbc/commit/97f3a4ba14ecb99b9bdfa03831a84cb7e1a8e970
+                    //     name = "Lance Norskog"
+                    //     email = "lance@edmodo.com"
+                    // }
+                    developer {
+                        name = "Roland Rifandi Utama"
+                        email = "roland_hawk@yahoo.com"
+                    }
```

Comment:
@rolandhawk Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-redshift` on Maven Central, from your work in #105. What do you think?

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.